scripts/apps/vocabularies/views/vocabulary-config-modal.html

```diff
@@ -18,7 +18,7 @@ <h3 class="modal__heading" ng-show="!vocabulary._id"><span translate>Create</spa
                                       ng-readonly="!!vocabulary._links"
                                       required>
                     <div class="sd-line-input__message"
-                         ng-show="issues._id.unique" translate>ID must be unique.</div>
+                         ng-show="issues._id.unique || issues._id.deleted" translate>ID must be unique.</div>
```

Comment:
Use error messages returned by API. `<div ng-repeat="issue in issues track by $index" class="sd-line-input__message">{{issue.message}}</div>`

Reply:
Do the same for a few fields below the one you added.

Reply:
Actually, the message in the API response is of the form `{"unique":1}` or `{"deleted":1}` or `{"conflict":1}`. So I think it won't be a good idea to display them as it is.

Reply:
Translation is not supported on the back-end yet, so we don't have a choice in this case. If it was, I would definitely want error messages returned from the server.

I've created `SDESK-3834` to enable translations on the server.